analyzer/src/funTest/kotlin/managers/SpdxDocumentFileFunTest.kt

```diff
@@ -202,7 +202,7 @@ class SpdxDocumentFileFunTest : WordSpec({
                 revision = vcsRevision
             )
 
-            val analyzer = Analyzer(AnalyzerConfiguration())
+            val analyzer = Analyzer(AnalyzerConfiguration(true))
```

Comment:
Please use a named argument when passing a boolean for readability.

Reply:
In Kotlin, [named arguments](https://kotlinlang.org/docs/functions.html#named-arguments) mean to write `allowDynamicVersions = true` directly in the list of arguments in line 205.

Reply:
@sschuberth thanks for the tip, corrected.